src/utils/did.js

```diff
@@ -185,3 +185,43 @@ export async function createSignedDidRemoval(didModule, did, currentKeyPair) {
   const signature = signDidRemoval(didModule, didRemoval, currentKeyPair);
   return [didRemoval, signature];
 }
+
+/**
+ * Create and return a `Attestation` as expected by the Substrate node. Signing is intentionally kept separate as the
+ * JS code may not have access to the signing key like in case of hardware wallet.
+ * @param {Number} priority - Attestation priority
+ * @param {*} iri - Attestation iri
```

Comment:
Would this work? Users need to be able to set attestation iri to None on-chain in order to remove their previous attestation.

```suggestion
 * @param {string | null} iri - Attestation iri
```

This may be out of scope for this pr so maybe best to just create a ticket for it.

Reply:
that should work i think, `any` type was left from when i was returning the object initially before our conversation